elasticsearch/src/main/java/org/apache/metamodel/elasticsearch/ElasticSearchUtils.java

```diff
@@ -44,8 +47,17 @@ public static Row createRow(Map<String, Object> sourceMap, String documentId, Da
                 values[i] = documentId;
             } else {
                 Object value = sourceMap.get(column.getName());
-                
-                values[i] = value;
+
+                if (column.getType() == ColumnType.DATE) {
+                    Date valueToDate = TimeComparator.toDate(value);
```

Comment:
Maybe it's better that we put in the ES module the time format that ES uses. I guess the TimeComparator is a bit more general purpose while I imagine (?) that ES is always using one specific date pattern.